DOCS-8448 JSON Web Tokens editor review (#162)
mulesoft/docs-api-community-manager: modules/ROOT/pages/install-validate.adoc

To verify your installation and configuration:

> . If you are using JSON Web Tokens and not impersonation, verify the Named Credentials.

Reviewer: ```suggestion
. If you are using JSON Web Token (JWT) and not impersonation, verify the named credentials.
```

.. Open *Setup*.

> .. Search for *Named Credentials*.

Reviewer: A search is for a lowercase string of syntax: ` `

> .. Locate *anypoint.mulesoft* and click *Edit*.

Reviewer: ditto previous ? for anypoint.mulesoft

> .. Verify that the list *External Data User Authentications* shows your administrative Anypoint user.

Reviewer: ```suggestion
.. Verify that the list *External Data User Authentications* shows your administrative user.
```

> . If you are using impersonation and not JSON Web Tokens, verify the Exchange External Data Source status is *Authenticated*.

Reviewer: ```suggestion
. If you are using impersonation and not JWT, verify that *Exchange External Data Source* status is *Authenticated*.
```

.. Open *Setup*.

> .. Search for *External Data Sources*.

Reviewer: ditto re. search text format
